pep-0570.rst

```diff
@@ -239,14 +252,15 @@ Improving Language Consistency
 The Python language, itself, would be more consistent with positional-only
 parameters. If the concept is a normal feature of Python rather than a feature
 exclusive to extension modules, it would reduce confusion for users
-encountering functions with positional-only arguments. Again, major
+encountering functions with positional-only arguments. Some major
```

Comment:
Since this is talking about the definition of the function, I think "parameters" should be used instead of "arguments".